packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue

```diff
@@ -1,62 +1,85 @@
 <template>
-	<div class="matrix-toolbar">
-		<Dropdown
-			v-if="matrixMap && Object.keys(matrixMap).length > 0"
-			:model-value="matrixType"
-			:options="matrixTypes"
-			placeholder="Select matrix type"
-			@update:model-value="(v) => changeMatrix(v)"
-		/>
-
+	<div class="toolbar">
+		<div v-if="matrixMap && Object.keys(matrixMap).length > 0">
+			<label>Measure</label>
+			<Dropdown
+				:model-value="matrixType"
+				:options="matrixTypes"
+				placeholder="Select matrix type"
+				@update:model-value="(v) => changeMatrix(v)"
+			/>
+		</div>
+		<div v-if="matrix.length > 1">
+			<label>Rows</label>
+			<MultiSelect
+				v-model="filteredRowNames"
+				:options="matrix.map((r) => r[0].rowCriteria)"
+				:max-selected-labels="2"
+				filter
+			/>
```

Comment:
This task ain't that easy ... 

Filter a matrix breaks copy-and-paste function.

For example, I stratify S such that I have:
- S_foo = 123
- S_bar = 456

If I filter to only show `S_bar` and paste in "999", S_foo is the one that is updated even though S_bar is the one that showing in the matrix. 

Also pasting seems to update the matrix but breaks the UI when the matrix-modal is dismissed, though I think this might be a separate issue - I will file a separate ticket and link here.

Before we start addressing this, check with Pascale and design folks to see if there are any additional requirements when pasting a grid of numbers while filtering is active.

Reply:
@pascaleproulx said that accommodating this would be a refinement. For now I can add a little warning message when some rows/cols are unselected to let the user know that paste may paste into rows/column you have hidden